content/kubermatic/master/examples/Deploy_your_application/_index.en.md

```diff
@@ -6,7 +6,7 @@ weight = 110
 
 This tutorial demonstrates how to deploy your application to your cluster. We will deploy a hello world application to your cluster that responds with "Hello Kubernetes!" when you curl it.
 
-Log into Kubermatic Kubernetes Platform (KKP) and [create a cluster](../03-create-cluster/), then [connect to the cluster](../06-download-kubeconfig/). For this tutorial you will need to see the external IP, be aware that some providers like DigitalOcean do not provide external IPs.
+Log into Kubermatic Kubernetes Platform (KKP), then [create and connect to the cluster](https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/project_and_cluster_management/). For this tutorial, external IP is required which will be provided through the supported cloud providers. Be aware that some providers like DigitalOcean do not provide external IPs.
```

Comment:
Is it necessary to use absolute path to the `tutorials_howtos/project_and_cluster_management/`
When we change the domain it will stop working